tools/run_wasm

```diff
@@ -41,7 +41,7 @@ while [ $# -gt 0 ]; do
       ;;
 
     --shell-option=*)
-      SHELL_OPTIONS+=("${1#--shell-option=}")
+      SHELL_OPTIONS+=" ${1#--shell-option=}"
```

Comment:
The `SHELL_OPTIONS` is an array, so I think the previous code was correct.

Reply:
Oops, you're right